config/nrfconnect/app/sample-defaults.conf

```diff
@@ -52,6 +52,9 @@ CONFIG_BT_DEVICE_NAME_MAX=15
 CONFIG_BT_MAX_CONN=1
 CONFIG_BT_PHY_UPDATE=n
 CONFIG_BT_CTLR_PHY_2M=n
+CONFIG_BT_GAP_AUTO_UPDATE_CONN_PARAMS=y
+CONFIG_BT_GAP_PERIPHERAL_PREF_PARAMS=y
+CONFIG_BT_CONN_PARAM_UPDATE_TIMEOUT=100
```

Comment:
As discussed, I propose to consider changing the preferred connection interval values from (24-40) to sth like (30-100), to reduce the probability of losing frames on Thread. Could you verify what is connection interval used by the Android phone by default?

Reply:
Sure, will do. If it comes to the Android phone (I assume you mean nRF Connect for Mobile not CHIPTool), it looks like it does something a bit more complex than mcumgr and master requests changing parameters few times on its own. Nevertheless it start with 7,5 ms interval like mcumgr, but immediately after GATT discovery switches to 45 ms interval.

Reply:
Fixed, after increasing connection intervals to 40ms-125ms, master accepts them and selects rather big interval of 120 ms instead of the smallest available. It results in quite slow DFU progress, however i don't think it is something that bad, as it will anyway happen in the application background.